communications/ff_msgs/action/Motion.action

```diff
@@ -111,9 +111,8 @@ ff_msgs/ControlState[] segment
 ff_msgs/MotionState state
 
 # Control progress
-{% if ROS1 %}
-ff_msgs/ControlFeedback progress                    # TODO(@trey0): ROS2 does not allow this
-{% endif %}
+# ff_msgs/ControlFeedback progress                    # ROS2 does not allow this
```

Comment:
Is this a todo for later? Seems like we may need it..

Reply:
The split between goal/feedback/result is not present anymore on ROS2, only a full action message. This is still a TODO that we need to either create a separate feedback message type or pass in the entire action message and only fill in the feedback portion. I'll add this back as TODO, thank you for noticing!